doc/nrf/libraries/bluetooth_services/services/hogp.rst

```diff
@@ -80,43 +70,58 @@ It can be obtained by calling :c:func:`bt_hogp_rep_size`.
-   This function iterates through all detected reports (excluding boot reports).
-   To find a specific report, use :c:func:`bt_hogp_rep_find`.
-   This function locates a report based on its type and ID.
+   .. group-tab:: Normal report
 
-Boot report:
-   If the connected device supports the boot protocol, it must have mouse and/or keyboard boot reports available.
-   This means that:
+      The report info pointer for a normal report can be retrieved with the :c:func:`bt_hogp_rep_next` function.
+      This function iterates through all detected reports (excluding boot reports).
+      To find a specific report, use :c:func:`bt_hogp_rep_find`.
+      This function locates a report based on its type and ID.
 
-   * For the mouse boot protocol, the function :c:func:`bt_hogp_rep_boot_mouse_in` returns a non-NULL value.
-   * For the keyboard boot protocol, the two functions :c:func:`bt_hogp_rep_boot_kbd_in` and :c:func:`bt_hogp_rep_boot_kbd_out` return a non-NULL value.
+   .. group-tab:: Boot report
 
-   All these functions return report pointers that may be used in the access functions.
-   Note, however, that these pointers cannot be used as a previous record pointer in :c:func:`bt_hogp_rep_next`.
+      If the connected device supports the boot protocol, it must have mouse and/or keyboard boot reports available.
+      This means that:
 
+      * For the mouse boot protocol, the function :c:func:`bt_hogp_rep_boot_mouse_in` returns a non-NULL value.
+      * For the keyboard boot protocol, the two functions :c:func:`bt_hogp_rep_boot_kbd_in` and :c:func:`bt_hogp_rep_boot_kbd_out` return a non-NULL value.
+
+      All these functions return report pointers that may be used in the access functions.
+      Note, however, that these pointers cannot be used as a previous record pointer in :c:func:`bt_hogp_rep_next`.
 
 Switching between boot and report mode
 ======================================
 
 To switch between Boot Protocol Mode and Report Protocol Mode, use :c:func:`bt_hogp_pm_write`.
+Every time the protocol mode is changed, the :c:type:`bt_hogp_pm_update_cb` function is called.
 
 You can retrieve the current protocol with :c:func:`bt_hogp_pm_get`.
 This function returns the internally cached version of the current protocol mode.
 To update this value directly from the device, use :c:func:`bt_hogp_pm_update`.
 
-.. note::
-   Every time the protocol mode is changed, the :c:type:`bt_hogp_pm_update_cb` function is called.
-
-
 Suspending and resuming
 =======================
 
 To suspend the connected device, call :c:func:`bt_hogp_suspend`.
 
-To resume, call :c:func:`bt_hogp_exit_suspend`.
+To resume, it call :c:func:`bt_hogp_exit_suspend`.
+
+Samples using the library
+*************************
+
+The following |NCS| samples use this library:
+
+* :ref:`hids_readme`
+
+The following |NCS| applications use this library:
+
+* :ref:`nrf_desktop_hid_forward`
+
+Dependencies
```

Comment:
Is this a mandatory section, if not we can skip this may be? If it is a mandatory section, are all the other RSTs in the PR having this?

Reply:
Good point, thank you Uma! I added missing sections added to other RST files.